tests/integration/test_migrate.sh

```diff
@@ -72,7 +72,7 @@ trap cleanup EXIT
 # at /var/lib/docker
 atomic_migrate () {
 	setup
-	${ATOMIC} migrate export --dir "$(pwd)/migrate-dir"
+	echo 'y'|${ATOMIC} migrate export --dir "$(pwd)/migrate-dir"
```

Comment:
It seems like there should be a new atomic flag e.g. `-y` or `--yes` so that these commands can be used non-interactively, like in these tests.